CONTRIBUTING.md

```diff
@@ -0,0 +1,244 @@
+
+Focus on giving constructive, actionable feedback that results in real improvements to code quality or end-user experience.
+If you don't understand why an approach was taken, please ask!
+
+Small changes work well as comments or in-line suggestions on specific lines of codes.
+Larger changes deserve a comment in the main thread, or a pull request to the original author's branch (but mention that you've made one).
+When in doubt about a matter of architectural philosophy, refer back to **What we're trying to build** for guidance.
+
+Once you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your `Approved` review on the PR so we can mark it as `ready-for-cart`.
+
+There are a two main places you can check for new work to review:
+
+1. Pull requests on `bevy` and the `bevy-website` repos.
+2. [RFCs](https://github.com/bevyengine/rfcs), which need extensive thoughtful community input on their design.
+
+Official focus areas and work done by @cart go through this review process as well.
+Not even our project lead is exempt from reviews and RFCs!
+By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely.
+
+Finally, if nothing brings you more satisfaction than seeing every last issue tagged and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
+As discussed in **How we're organized**, this is not intended to have a high bar.
+
+### Contributing your own ideas
+
+As discussed in **How we work together** Bevy is actively open to new ideas and serious contributions from outside community members.
+If you're new to Bevy, here's the workflow we use:
+
+1. Fork the `bevyengine/bevy` repository on GitHub, you'll need to create a GitHub account if you don't have one already.*
+2. Make your changes in a local clone of your fork, typically in its own new branch.
+3. For a higher chance of CI passing the first time, consider locally running `cargo run -p ci`. You can run the commands manually:
+    1. `cargo fmt --all -- --check` (remove `--check` to let the command fix found problems)
+    2. `cargo clippy --workspace --all-targets --all-features -- -D warnings -A clippy::type_complexity`
+    3. `cargo test --all-targets --workspace`
+4. Push your changes to your fork and open a Pull Request.
+5. If you're a first time contributor to this repo, @cart [will need to manually trigger CI for your PR](https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs/). Feel free to ping him for this.
+6. Respond to any CI failures or review feedback.
+
+If you end up creating a new crate to the `bevy` repo:
+
+1. Add a "Bevy Contributors <bevyengine@gmail.com>" entry in the Author field of `Cargo.toml`.
+2. Add an MIT License to match the main `bevy` crate.
```

Comment:
This is actually not the case at the moment. Do we want/need this? I guess it would make a lot of sense at least in cases like `bevy_ecs` where we explicitly say that they can be used standalone.

Reply:
Hmm, good question 🤔 I think it's clearer, but it depends on what shows up on e.g. docs.rs.

Reply:
Okay docs.rs correctly identifies the license as being inherited. Great, I'm removing this license.